packages/relay/tests/lib/config/hbarSpendingPlanConfigService.spec.ts

```diff
@@ -502,6 +520,107 @@ describe('HbarSpendingPlanConfigService', function () {
         });
       });
     });
+
+    describe('Environment variable defined spending plans', function () {
+      const spendingPlansConfigExample = [
```

Comment:
For better type checking:
```suggestion
      const spendingPlansConfigExample: SpendingPlanConfig[] = [
```

Reply:
Actually, we already have this available in the `spendingPlansConfig` variable above:

```typescript
  const spendingPlansConfigFile = 'spendingPlansConfig.example.json';
  const path = findConfig(spendingPlansConfigFile);
  const spendingPlansConfig = JSON.parse(fs.readFileSync(path!, 'utf-8')) as SpendingPlanConfig[];
```

Reply:
You resolved this comment, but you still haven't removed this constant and reused `spendingPlansConfig`, could you explain?

Reply:
Removed and reused existing tests as suggested.